docs/advanced-curriculum.adoc

```diff
@@ -33,13 +33,13 @@ endif::[]
 :sectnums!:
 include::00-preamble/copyright.adoc[{include_configuration}]
 
-<<<
-toc::[]
```

Comment:
Can we move the TOC back to this place? Other curricula also start with the TOC directly after the copyright.

Reply:
I understand, there is no common ground for this so far, so we are defining it now. Will revert